src/controller/CommissioningDelegate.h

```diff
@@ -414,6 +414,7 @@ class CommissioningParameters
     CompletionStatus completionStatus;
     Credentials::DeviceAttestationDelegate * mDeviceAttestationDelegate =
         nullptr; // Delegate to handle device attestation failures during commissioning
+    Optional<bool> mShouldWaitPostDeviceAttestation;
```

Comment:
Expose methods to set and get these?

Reply:
I actually meant to remove this entirely, because the ShouldWaitPostDeviceAttestation question is answered by the delegate object.